libraries/MySensors/examples/SensebenderMicro/SensebenderMicro.ino

```diff
@@ -20,6 +20,9 @@
 // Define a static node address, remove if you want auto address assignment
 //#define NODE_ADDRESS   3
 
+// Uncomment the line below, to transmit battery voltage as a normal sensor value
+//#define BATT_SENSOR    199
```

Comment:
This is specific for the sensebender micro...
Den 07/06/2015 22.36 skrev "Tomas Hozza" <notifications@github.com>:

> In libraries/MySensors/examples/SensebenderMicro/SensebenderMicro.ino
> <https://github.com/mysensors/Arduino/pull/122#discussion_r31880337>:
>
> > @@ -20,6 +20,9 @@
> >  // Define a static node address, remove if you want auto address assignment
> >  //#define NODE_ADDRESS   3
> >
> > +// Uncomment the line below, to transmit battery voltage as a normal sensor value
> > +//#define BATT_SENSOR    199
>
> Why didn't you add this to the MyMessage.h?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mysensors/Arduino/pull/122/files#r31880337>.
>